### DISCUSSION TITLE
[vite] no tiles loaded

### DISCUSSION THREAD

**Discussion by kevinrss01** [iTwin/itwinjs-core#5433] [Q&A] [answered]
Hello,

I am switching my Itwin React application to Vite/React, the problem is that when I get to the home page I have a black screen instead of the 3D model. Does anyone have any idea where that might have come from?

![image](https://user-images.githubusercontent.com/94388367/233591876-9785537c-bd0d-4f55-9cbf-1399e8b36dfb.png)

You can find my project here : https://github.com/kevinrss01/app-itwin-and-vite/tree/main

**Comment by pmconne** [comment]
Check browser console for error messages and network tab for failing requests.

**Comment by aruniverse** [comment, marked as answer]
Take a look at https://github.com/iTwin/itwinjs-core/discussions/5185#discussioncomment-5266617

**Reply by kevinrss01** [reply]
Thanks a lot! It solved my issue.
